exec/java-exec/pom.xml

```diff
@@ -309,7 +309,7 @@
     <dependency>
       <groupId>org.apache.curator</groupId>
       <artifactId>curator-x-discovery</artifactId>
-      <version>2.7.1</version>
+      <version>${curator.version}</version>
```

Comment:
`<version>` and `<exclusions>` aren't necessary, if you specified them in the dependencyManagement block